Move pages.tx_templavoila_flex to own tab, instead of cluttering tab 'general'

jpmschuler opened the pull request:
In a recent design I got a page datastructure which has a whole lot of tt_content fields. Thus, while editing a page datarow, the pages.general tab gets cluttered. Normally I won't mind that, but the current position of the fields is quite early. That made using the page types "shortcut" and "external url" difficult, because their url target fields were below the pages.tx_templavoila_flex, thus not even on my screen. Editors find that annoying. Although that should be solved by the core team moving the url target fields next to the page type, I consider an easier approach:

**Assumptions:**
- adding pages.tx_templavoila_flex at a new position "last element of general tab" is quite impossible hard due to all the page types (and page types being different in different LTS versions)
- the field should probably be never changed in "edit pages datarow" anyways, but instead in TV+ page module, so it could be deemed as not that important and thus shouldn't be that prominent

**Suggestion:** 
Thus, I suggest to move them to an own tab, at the end of the list of tabs.
<img width="856" alt="WIWI-chairT3_dev_System_vom_2019-04-07__TYPO3_CMS_8_7_24_" src="https://user-images.githubusercontent.com/12411176/55689785-25bc4d80-5989-11e9-8b7e-d8f3f2b1d921.png">

opi99 commented:
For 8.0 or should we add this also in 7.3.1?

jpmschuler commented:
¯\_(ツ)_/¯ I'm just proposing this as a general improvement - based on the kind of change this should be compatible to all LTS versions.
As I use composer with `cweagans/composer-patches` I auto-hoffix all my installs anyways and am never in a hurry to get a feature into the codebase.
But perhaps this could improve understandability of integrators also, which might prevent things like #233.

fgerards commented:
furthermore, there should be a way to split FCEs in diff. custom Tabs in the NCEW. We achieved this by reproducing the URL to each FCE with TSConfig, but that is not really the way to go I guess ;)

jpmschuler commented:
squashed the commits and rebased them for 

- master/8.x here in this PR
- and for legacy 7.3.x usage in https://github.com/jpmschuler/templavoilaplus/pull/11.patch (e.g. with https://github.com/cweagans/composer-patches)
